importer.js

```diff
@@ -6,36 +6,32 @@ const transform = require('stream-transform');
 
 
 function parseValue(recordValue, mappingObject) {
-  if(mappingObject.format === 'jsDate') {
+  if(mappingObject === 'timestamp') {
     // convert millisconds to nanoseconds
     return (new Date(recordValue)).getTime() * 1000 * 1000;
   }
   return recordValue;
 }
 
-function flatMappingToInfuxFieldSchema(mapping) {
+function flatMappingToInfluxFieldSchema(mapping) {
   var res = {};
   var namesMapping = {};
-  for(var k in mapping.fieldSchema) {
-    if(k === mapping.timestamp) {
-      res['time'] = 'timestamp';
-    } else {
-      var distName = k;
-      if(typeof mapping.fieldSchema[k] === 'string') {
-        res[k] = mapping.fieldSchema[k];
-      } else {
-        if(mapping.fieldSchema[k].type === undefined) {
-          console.error('mapping.fieldSchema[' + k + '].type is undefined');
-          process.exit(errors.ERROR_BAD_CONFIG_FORMAT);
-        }
-        if(mapping.fieldSchema[k].name !== undefined) {
-          distName = mapping.fieldSchema[k].name;
-        }
-        res[distName] = mapping.fieldSchema[k].type;
-      }
-      namesMapping[k] = distName;
+  var schema = mapping.fieldSchema;
+
+  Object.keys(schema).forEach(key => {
+    if(schema[key].type === undefined) {
+      console.error('mapping.fieldSchema[' + key + '].type is undefined');
+      process.exit(errors.ERROR_BAD_CONFIG_FORMAT);
     }
-  }
+    if(schema[key].from === undefined) {
```

Comment:
to Docs as well.

Reply:
Added at https://github.com/CorpGlory/csv2influx/pull/13/commits/72aea16105d925b4ac55ccd37533b1dbb7d9543a as well